translations/base-de.yaml

```diff
@@ -32,39 +35,56 @@ steamPage:
 
         In shapez.io musst du Maschinen geschickt verbinden, damit Formen automatisiert erstellt, bearbeitet und kombiniert werden. Liefere die gewünschten, stetig komplexer werdenden Formen an dein Hauptgebäude, um im Spiel voranzukommen. Schalte mit ihnen außerdem Upgrades frei, die deine Maschinen und somit auch deine Fabriken beschleunigen!
 
-        Da die Nachfrage sowohl in der Komplexität, als auch der Anzahl steigt, wirst du deine Fabriken erweitern müssen. Vergiss nicht, dass du die dafür benötigten Ressourcen beschaffen musst und expandiere auf der [b]unendlichen Karte[/b]!
+        Da die Nachfrage sowohl in der Komplexität, als auch der Menge steigt, wirst du deine Fabriken erweitern müssen. Vergiss nicht, dass du die dafür benötigten Ressourcen beschaffen musst und expandiere auf der [b]unendlichen Karte[/b]!
 
-        Das reine Zerschneiden und Zusammensetzen von Formen kann natürlich schnell langweilig werden, aber keine Sorge - Du kannst Farben mischen und deine Formen damit bemalen lassen. Staple dann deine fertigen Formen aufeinander und lasse so die wildesten Kreationen entstehen.
+        Bald wirst du Farben mischen und deine Formen damit bemalen lassen. Staple dann deine fertigen Formen aufeinander und lasse so die wildesten Kreationen entstehen.
 
         Nutze dein gesammeltes Wissen über die Maschinen und lasse deine Fabriken die gewünschten Formen der 18 verschiedenen Level abliefern. Schalte mit jedem Level neue Arbeitsschritte oder Gebäude frei. Das sollte dich schon für Stunden beschäftigt halten! Danach werden im Freispielmodus zufällige Formen generiert, die du ebenfalls abliefern kannst. Ich füge regelmäßig neue Funktionen hinzu und davon sind eine ganze Menge geplant!
+        
+        Wenn du das Spiel erwirbst, erhälst du Zugriff auf die zusätzlichen Features der Standalone-Version. Das bedeutet, du kannst unter anderem die neuesten Updates zuerst spielen!
 
-        [b]Vorteile der Standalone[/b]
+        [h1]VORTEILE DER STANDALONE[/h1]
 
         [list]
-            [*] Wegpunkte
-            [*] Unbegrenzte Anzahl von Spielständen
             [*] Dark-Mode
+            [*] So viele Wegpunkte wie du willst
+            [*] Unbegrenzte Anzahl von Spielständen
             [*] Weitere Einstellungen
+            [*] Bald erhältlich:
```

Comment:
I would like to get some Feedback on this translation. Another option might be `Neue Funktionen in der nahen Zukunft` which sounds clunky for now but might be more accurate. Suggestions are welcome